src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -507,11 +507,20 @@
         "3.1.0",
         "3.1.1",
         "3.1.2",
-        "5.0.0"
+        "5.0.0",
+        "3.1.3",
+        "3.1.4",
+        "3.1.5",
+        "3.1.6",
+        "3.1.7",
+        "3.1.8",
+        "3.1.9",
+        "3.1.10"
```

Comment:
nit: lets sort this

Reply:
Will do. It looked odd to me too. but I left as is since it was auto generated.